.github/workflows/ci.yml

```diff
@@ -35,22 +41,24 @@ jobs:
         run: yarn install
 
       - name: lint
-        run: yarn lint
+        run: yarn workspace ${{ matrix.workspace }} lint
```

Comment:
I'm wondering if we should run the lint for all workspaces in just one job instead

Reply:
But this way we can run the jobs in parallel?

Reply:
yeah, but it has to perform the `yarn install` for every single job which most likely takes longer than the actual linting

Reply:
The linting jobs still finish in around 1m though: https://github.com/simplabs/ember-simple-auth/pull/2184/checks?check_run_id=608604147 which is substantially faster than the jobs than run tests. Assuming all jobs run in parallel I think keeping the CI config simpler trumps speeding up these jobs that are not a limiting factor for the CI pipeline anyway?